### PR TITLE
enh(mc): automatic commit for network-switchs-juniper-ex-snmp

### PR DESCRIPTION
**Pull request by omercier**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 1]
```diff
@@ -2,48 +2,633 @@
...
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="Net-Juniper-EX-custom" label="Net-Juniper-EX-custom">
+
+| Alias          | Modèle de service                    | Description                                        |
+|:---------------|:-------------------------------------|:---------------------------------------------------|
+| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Contrôle l'utilisation CPU du 'routing engine'     |
+| Hardware       | Net-Juniper-Ex-Hardware-custom       | Contrôle l'état du matériel                        |
+| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'Routing Engine' |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:26]:
```suggestion
| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Contrôle l'utilisation mémoire du 'routing engine' |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 2]
```diff
@@ -2,48 +2,633 @@
...
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resources through an SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-EX-custom** host template |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle              | Description                                                             |
+|:-----------------------------|:------------------------------------------------------------------------|
+| Net-Juniper-Ex-Storages-Name | Discover the disk partitions and monitor space occupation               |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:63]: why one in English and one in French?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 3]
```diff
@@ -2,48 +2,633 @@
...
+| Net-Juniper-Ex-Traffic-Name  | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:64]:
```suggestion
| Net-Juniper-Ex-Traffic-Name  | Découvre les interfaces réseau et supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 4]
```diff
@@ -2,48 +2,633 @@
...
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
 
-### Configure SNMP on your server
+<Tabs groupId="sync">
+<TabItem value="Cpu-Routing" label="Cpu-Routing">
 
-Follow constructor procedure for your equipment.
+| Macro           | Description                                                                                        | Valeur par défaut | Obligatoire |
+|:----------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTER          | Filter operating (default: 'routing\|fpc')                                                         | routing           |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:252]: filter operating what?

File: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 5]
```diff
@@ -2,48 +2,632 @@
...
+* **Net-Juniper-EX-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Net-Juniper-EX-custom" label="Net-Juniper-EX-custom">
+
+| Service Alias  | Service Template                     | Service Description                  |
+|:---------------|:-------------------------------------|:-------------------------------------|
+| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU Usage of routing engine    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:23]:
```suggestion
| Cpu-Routing    | Net-Juniper-Ex-Cpu-Routing-custom    | Check CPU usage of routing engine    |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 6]
```diff
@@ -2,48 +2,632 @@
...
+| Hardware       | Net-Juniper-Ex-Hardware-custom       | Check hardware                       |
+| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check Memory Usage of routing engine |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:25]:
```suggestion
| Memory-Routing | Net-Juniper-Ex-Memory-Routing-custom | Check memory usage of routing engine |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 7]
```diff
@@ -2,48 +2,632 @@
...
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias        | Service Template                           | Service Description                                                                          | Discovery  |
+|:---------------------|:-------------------------------------------|:---------------------------------------------------------------------------------------------|:----------:|
+| Disk-Generic-Id      | Net-Juniper-Ex-Disk-Generic-Id-custom      | Check the rate of free space on the disk                                                     |            |
+| Disk-Generic-Name    | Net-Juniper-Ex-Disk-Generic-Name-custom    | Check the rate of free space on the disk                                                     |            |
+| Disk-Global          | Net-Juniper-Ex-Disk-Global-custom          | Check the rate of free space on the disk                                                     | X          |
+| Traffic-Generic-Id   | Net-Juniper-Ex-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:37]:
```suggestion
| Traffic-Generic-Id   | Net-Juniper-Ex-Traffic-Generic-Id-custom   | Check the bandwidth of the interface. For each check the name of the interface will appear  |            |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 8]
```diff
@@ -2,48 +2,632 @@
...
+| Traffic-Generic-Name | Net-Juniper-Ex-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each checks the name of the interface will appear  |            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:38]:
```suggestion
| Traffic-Generic-Name | Net-Juniper-Ex-Traffic-Generic-Name-custom | Check the bandwidth of the interface. For each check the name of the interface will appear  |            |
```

File: pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md [hunk 9]
```diff
@@ -2,48 +2,632 @@
...
+| Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each checks the name of the interface will appear    | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:39]:
```suggestion
| Traffic-Global       | Net-Juniper-Ex-Traffic-Global-custom       | Check the bandwidth of  interfaces. For each check the name of the interface will appear    | X          |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md (hunk 4 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:252]:
```suggestion
| FILTER          | Define which elements should be monitored based on their name. This option will be treated as a regular expression. 'routing\|fpc' is applied if left empty.   | routing           |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md (hunk 2 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:63]:
```suggestion
| Net-Juniper-Ex-Storages-Name | Découvre les disques pour en faciliter la mise en supervision.               |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md (hunk 1 above)
Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-switchs-juniper-ex-snmp.md:26]: In that case you should have the capital letters everywhere and not just on that line

**Review by cg-tw — APPROVED** [review]

**Pull request merged**